src/react/6-stateful-hooks.md

```diff
@@ -1,28 +1,28 @@
 @page learn-react/stateful-hooks Stateful Hooks
 @parent learn-react 6
 
-@description Learn how to use hooks in React to manage the state of a component.
+@description Learn how to use React's hooks to manage the state of a component.
```

Comment:
The more common nomenclature is "React Hooks", not "React's hooks".

From the official docs:

![image](https://user-images.githubusercontent.com/10255546/100479796-42a51780-30bd-11eb-850d-3c34edecfd6c.png)

Reply:
Done